chaincode/src/fees/exitGateImplementations.ts

```diff
@@ -86,7 +93,7 @@ export async function mintPostProcessing(ctx: GalaChainContext, data: IMintPostP
 export interface ILockOnMintProcessing {
   tokenClass: TokenClassKey;
   tokens: TokenInstanceKey[];
-  owner: UserAlias;
+  owner: string;
```

Comment:
can it be `UserRef`? Typically we want to use `UserRef` for input, and `UserAlias` where we want the data to be unambiguous.

if you need, at some point transform a `string` to `UserRef`, you may use `asValidUserRef` function, which performs validation

Reply:
This was my oversight, I think this change to `string` came through while I was reworking and getting more familiar with the `UserRef` vs `UserAlias` api. It's fixed now in a new commit.